programs/server/config.xml

```diff
@@ -96,7 +96,7 @@
             <to>https://{bucket}.s3.amazonaws.com</to>
         </s3>
         <gs>
-            <to>https://{bucket}.storage.googleapis.com</to>
+            <to>https://storage.googleapis.com/{bucket}</to>
```

Comment:
So this was a mistake and `https://{bucket}.storage.googleapis.com` is always wrong?

Reply:
yes, `https://{bucket}.storage.googleapis.com` is always wrong

> As the issue can be fixed with a config change, should we recommend the user to do it until the fix lands?

- user already has a solution for this problem, as he uses a direct link instead s3-style link

Reply:
I think it still makes sense to make them aware that they can modify config if they still want to use s3-style links.